src/main/java/controllers/ConsoleInputController.java

```diff
@@ -221,6 +222,46 @@ public void onCommandReceived(String input) {
                 consoleView.consolePrint("Project created!");
             }
             break;
+        case "manage":
+            if (inputReader.hasNext()) {
+                int projectNumber = Integer.parseInt(inputReader.next());
+                IProject projectToManage = projectRepository.getProject(projectNumber);
+                String projectName = projectToManage.getDescription();
+                consoleView.consolePrint("Now managing: " + projectName);
+                //branch into project and begin to read commands to manage project
+                Scanner manageProjectInput = new Scanner(System.in);
+
+                boolean continueManaging = true;
+                while (continueManaging) {
+                    if (manageProjectInput.hasNextLine()) {
+                        String projectCommand = manageProjectInput.nextLine();
+                        switch (projectCommand) {
+                        case "exit":
+                            continueManaging = false;
+                            consoleView.exitProject(projectName);
+                            break;
+                        case "add member":
+                            consoleView.consolePrint("Enter member details: n/NAME p/PHONE e/EMAIL");
+                            String memberDetails = manageProjectInput.nextLine();
+                            Member newMember = MemberFactoryUtil.createMember(memberDetails);
+                            consoleView.addMember(projectToManage, newMember);
+                            break;
+                        case "view members":
+                            consoleView.viewAllMembers(projectToManage);
+                            break;
+                        default:
+                            consoleView.consolePrint("Invalid command. Try again!");
+                            break;
+                        }
+
+                    } else {
+                        consoleView.consolePrint("Please enter a command.");
+                    }
```

Comment:
Abstract to another controller to manage inputs for a specific project.

Reply:
Yeah I was thinking of doing that too!